toolkit/analysis_and_visualization/classification/assets/scripts/charts.js

```diff
@@ -0,0 +1,415 @@
+function sum(a, b) {
+    return a + b;
+}
+
+function drawChart() {
+
+    var chartData = google.visualization.arrayToDataTable([
+        ['  ', 'Match', 'Mismatch', 'No Label'],
+        ['Summary', data.stats.passCount, data.stats.totalMismatch, data.stats.totalNoGroundTruth]
```

Comment:
Codacy found an issue: ['data' is not defined.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6397592)